preprocessing/README.md

```diff
@@ -1,14 +1,14 @@
 Preprocessing
 =============
 
-This folder contains all (1) tools required to set a up the input and generated source files required by SeisSol.
+This folder contains all<sup>1</sup> tools required to set a up the input and generated source files required by SeisSol.
```

Comment:
reStructuredText has built-in support for foot-notes:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#footnotes

Moreover, the footnote is a contradiction... wouldn't it make more sense to replace "all" by "most" and list in the footnote what is missing?

Reply:
Forget about the reStructuredText comment... just noticed this is a markdown file.